README.md

```diff
@@ -91,10 +91,10 @@ const core = new CirclesCore(web3, {
   proxyFactoryAddress: '0x9b1f7F645351AF3631a656421eD2e40f2802E6c0',
   safeMasterAddress: '0x59d3631c86BbE35EF041872d502F218A39FBa150',
   apiServiceEndpoint: 'http://api.circles.local',
-  pathfinderServiceEndpoint: 'http://pathfinder.circles.local'
-  pathfinderType: 'server',
+  pathfinderServiceEndpoint: 'http://pathfinder.circles.local',
+  pathfinderType: 'server', // other option: 'cli'
   graphNodeEndpoint: 'http://graph.circles.local',
-  databaseSource: 'graph',
+  databaseSource: 'graph', // other options:
```

Comment:
No other options available yet. Adding the comment for `databaseSource` seems confusing

Reply:
ok but maybe revert the last commit and still fix the comma, since the code formatting in readme is wrong

Reply:
I am just lost on how to make the latest core version work with the old api

Reply:
Ok, then setting the pathfinder type to "cli" is enough